src/mumble/ConnectDialog.cpp

```diff
@@ -1467,20 +1468,21 @@ void ConnectDialog::startDns(ServerItem *si) {
 
 	QString host = si->qsHostname.toLower();
 
-	if (si->qlAddresses.isEmpty()) {
+	if (si->qlAddressPorts.isEmpty()) {
 		QHostAddress qha(si->qsHostname);
-		if (! qha.isNull())
-			si->qlAddresses.append(qha);
-		else
-			si->qlAddresses = qhDNSCache.value(host);
+		if (! qha.isNull()) {
```

Comment:
I would like to see a comment here. We check if it's an IP (rather than a hostname/domain) and use that if it is.
Or should I have known QHostAddress is for IP addresses (only)?